### PR TITLE
Add component for displaying full person details

### LINKED ISSUE #69: Display full contact details
As a user, I can see the full details of a contact.

### PR DESCRIPTION
**Pull request by chesterhow**
### Changes
- Created new UI component `PersonDetailsCard` to display the full details of a contact on the sidebar
- Improved styling of the profile picture on the `PersonListCard` component

Fixes #69

### CODE REVIEW

File: src/main/java/seedu/address/ui/PersonDetailsCard.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        name.setText(person.getName().fullName);
+        phone.setText(person.getPhone().value);
+        address.setText(person.getAddress().value);
+        email.setText(person.getEmail().value);
+        birthday.setText(person.getBirthday().toUi());
+        person.getTags().stream()
+                .sorted(Comparator.comparing(tag -> tag.tagName))
+                .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+
+        // Temporary UI to test dates and meetings
```
Comment (ivantjh) [inline src/main/java/seedu/address/ui/PersonDetailsCard.java:54]: Is this still temporary?

File: src/main/java/seedu/address/ui/PersonDetailsCard.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        person.getDates().forEach(date -> dates.getChildren().add(new Label(date.toString())));
```
Comment (ivantjh) [inline src/main/java/seedu/address/ui/PersonDetailsCard.java]: Should this be `date.toUi()`?

File: src/main/java/seedu/address/ui/PersonDetailsCard.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        person.getMeetings().forEach(meeting -> meetings.getChildren().add(new Label(meeting.toUi())));
+
+        Optional<Picture> personPicture = person.getPicture();
+        if (personPicture.isPresent()) {
+            File imgFile = new File(personPicture.get().getAbsoluteFilePath());
+            try {
+                Image userImage = new Image(new FileInputStream(imgFile));
```
Comment (ivantjh) [inline src/main/java/seedu/address/ui/PersonDetailsCard.java]: Possible to create another component to store the profile picture? The same code is used in `PersonDetailsCard` and `PersonCard`.

**Review by ivantjh — APPROVED** [review]
lgtm!

File: src/main/java/seedu/address/ui/PersonDetailsCard.java (hunk 1 above)
Comment (chesterhow) [reply src/main/java/seedu/address/ui/PersonDetailsCard.java:54]: yeah its just a label now no styling yet

**Pull request merged**